lib/timers.js

```diff
@@ -83,10 +83,19 @@ function listOnTimeout() {
   debug('timeout callback ' + msecs);
 
   var now = Timer.now();
-  debug('now: ' + now);
+  debug('now: %d', now);
```

Comment:
```debug('now: %d', now);``` Is using %d appropriate? or should we use %s, now can also have double value right?